### PR TITLE
Update RTC's Type

### PR DESCRIPTION
**Pull request by goetzrrGit**
I updated the `immediate_load` and `immediate_activate` types to be specific

ex.
```json
immediate_load{
        "args": {
          "$ref": "#/$defs/args"
        },
        "description": {
          "$ref": "#/$defs/description"
        },
        "engine": {
          "type": "integer",
          "description": "Sequence target engine."
        },
        "type": {
          "const": "load" <------------------------ making this 'immediate_load'
        }
}
```

Also for consistency I added `model` attribute from `immediateFSWCommands`. I verified with Taifun, Carter, and Mike Schaffer of these changes and they were onboard with it.

### CODE REVIEW

**Review by cohansen — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nasa-jpl/seq-json-schema",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (cartermak) [inline package.json]: I think this merits at least a minor version increment instead of just a patch, given we're changing the `type` values.

**Review by cartermak — CHANGES_REQUESTED** [review]
LGTM other than version increment.

File: package.json (hunk 1 above)
Comment (goetzrrGit) [reply package.json]: Sure, I made it 1.2.0

**Review by cartermak — APPROVED** [review]

**Pull request merged**